--- dist-packages/dotty_dict/dotty_dict.py	2019-03-17 05:01:08.000000000 +0100
+++ dist-packages/dotty_dict/dotty_dict.py-E	2019-03-17 05:07:06.000000000 +0100
@@ -39,7 +39,7 @@
     """
 
     def __init__(self, dictionary, separator, esc_char):
-        if not isinstance(dictionary, Mapping):
+        if not isinstance(dictionary, dict):
             raise AttributeError('Dictionary must be type of dict')
         else:
             self._data = dictionary
